__tests__/app.test.js

```diff
@@ -78,12 +78,56 @@ describe("/api", () => {
                     const ArtKeys = Object.keys(article[0])
                     expect(Object.keys(article).length).toBeGreaterThan(0)
                     expect(ArtKeys).toEqual(expect.arrayContaining(["author","title","article_id","body","topic","created_at","votes","article_img_url"]))
+                    expect(article[0].article_id).toBe(1)
                 })
             })
-            test("Returns 404 with invalid ID", () => {
+            test("Returns 404 \"Not Found\" with invalid ID", () => {
                 return request(app)
                 .get("/api/articles/20000")
                 .expect(404)
+                .then(({body}) => {
+                    expect(body.msg).toBe("Not Found")
+                })
+            })
+            test("Returns 400 \"Bad Request\" with non-existant ID", () => {
+                return request(app)
+                .get("/api/articles/notanumber")
+                .expect(400)
+                .then(({body}) => {
+                    expect(body.msg).toBe("Bad Request")
+                })
+            })
+        })
+        describe("articles/:article_id/comments", () => {
+            test("returns object with correct keys", () => {
+                return request(app)
+                .get("/api/articles/1/comments")
+                .expect(200)
+                .then(({body}) => {
+                    const { comments } = body
+                    expect(comments.length).toBeGreaterThan(0)
+                    comments.forEach(comment => {
+                    const commentKeys = Object.keys(comment)
+                    expect(commentKeys).toEqual(expect.arrayContaining(["comment_id","votes","created_at","author","body","article_id"]))
+                    expect(comment.article_id).toBe(1)
```

Comment:
This kind of assertion is fine as it's checking if the object got all the properties.

 I also recommend the .objectContaining and .toMatchObject jest matchers on the objects with the use of expect.any() which can check the type of the values as well.